source/contracts/reporting/Market.sol

```diff
@@ -115,28 +117,35 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
     }
 
     function doInitialReport(uint256[] _payoutNumerators, bool _invalid, string _description) public returns (bool) {
+        doInitialReportInternal(msg.sender, _payoutNumerators, _invalid, _description);
+        return true;
+    }
+
+    function doInitialReportInternal(address _reporter, uint256[] _payoutNumerators, bool _invalid, string _description) public returns (bool) {
+        require(!universe.isForking());
```

Comment:
Note: you can't report when a universe is forking or else anyone can come along and un-do your initial report.